spec/person_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../book'
+require_relative '../persons'
+
+describe Person do
+  context 'containing a list of names and ages' do
+    person = Person.new(45, 'Margaret')
+
+    it 'expects Margaret age to be 45' do
+      expect(person.age).to eq 45
+    end
+    it 'expects Person.name to be Margaret' do
+      expect(person.name).to eq 'Margaret'
+    end
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/217905427-8503219e-8f08-4da7-ba7d-fe384e4b1b60.png)

- [ ] You didn't test the correct_name method shown in the image above :thinking: Kindly test it :+1: to fully test your person class :blush: